### ISSUE TITLE
[Humanize AI Text] ...

### ISSUE THREAD

**Issue by PawGangsta** [raycast/extensions#12435]
### Extension

https://www.raycast.com/frolik/ai-humanizer

### Raycast Version

v1.74.0

### macOS Version

14.4.1

### Description

I installed the extension and I wanted to test it out so I copied AI text from bing, pasted where it says Original text, but "the humanized" section never shows up, even tho, it says that my text has been humanized or something like that, but I cant see it anywhere

### Steps To Reproduce

I don't know, write a blog on bing and try to humanize it, for the humanizing part (as shown in the screenshots) doesn't show up for me.

**Comment by frolvanya** [comment]
Yep, I used API from a random website and I guess they changed their API. I'll decide whether to address this issue on the next few days or remove extension from Raycast store

**Comment by frolvanya** [comment]
@raycastbot assign me

**Comment by PawGangsta** [comment]
@frolvanya If I may, I'd urge you to address the issue when your time allows it, because your extension is AWESOME, it would be such a loss

**Comment by frolvanya** [comment]
@PawGangsta Thanks for kind words! I've already fixed an issue in a recent PR, let's wait until it will be merged

**Comment by PawGangsta** [comment]
@frolvanya great news, can't wait to try it out :)
